app/components/reports/cohort_component.html.erb

```diff
@@ -0,0 +1,116 @@
+<div class="card w-100pt mt-lg-0">
+  <div class="mb-24px mb-lg-24px">
+    <h3>
+      <%= cohort_report_type(period) %> cohort
+    </h3>
+    <% if period.type == :month %>
+      <p class="mb-0px c-grey-dark c-print-black">
+        The result for all assigned hypertensive patients registered in a month at their follow-up visit in the following two months
```

Comment:
[Suggestion] Should we move these strings into `en.yml`?

Reply:
We don't have plans to translate the dashboard (verrrrry unlikely) so I'd leave it in here like the rest of our dashboard, avoid the indirection.